### ISSUE TITLE
root-signing-staging maintainer permissions not working

### ISSUE THREAD

**Issue by jku** [sigstore/community#356]
After a few attempts the pulumi update from merging https://github.com/sigstore/community/pull/355 (https://github.com/sigstore/community/actions/runs/6656388104) succeeded, so the root-signing-staging project permissions should now be good...

Unfortunately I'm not seeing that:
* I don't seem to have even triage permissions on the GitHub project as I can't label or assign issues
* I can't push a branch to the project

The yaml files in this project seem correct to me but I think I have no visibility to how pulumi sees them. There's some evidence towards the `tuf-root-signing-staging-codeowners` team being setup fine but my membership in the team not being applied:
* cpanato mentions in the discussion below that the team has 2 members
* kommendorkapten said he does have permissions in the project
* there should actually be 3 members in the team (see https://github.com/sigstore/community/blob/main/github-sync/github-data/sigstore/users.yaml)

So my guess is that the project and team are fine but my membership is somehow not being processed (even though similar memberships for kommendorkapten and haydentherapper are handled).

**Comment by jku** [comment]
Fredrik mentioned that he does have the permissions in the project so the issue seems to be my membership in the team?

**Comment by cpanato** [comment]
those are the teams in the root signing staging repo 
![Screenshot 2023-10-31 at 16 25 36](https://github.com/sigstore/community/assets/4115580/052b5149-e906-448e-9ec4-97213aced1ce)

you should not able to push the main branch, only via PR

**Comment by jku** [comment]
tuf-root-signing-staging-codeowners should have 3 members (me, haydentherapper and kommendorkapten). Looks like my membership has an issue, but I can't figure out what it is...

**Comment by kommendorkapten** [comment]
Would it be possible to see who is the members of `tuf-root-signing-staging-codeowners`? I would guess me and Hayden.

**Comment by jku** [comment]
Apparently the issue was that I had not been invited to sigstore org. @bobcallaway invited me now, I've accepted, and I've now been added to the team as well.

I'm going to close this. I think the root cause was:
* I did not realize that adding a user to the config in this project does not add that user to the org
* My PR #348 ended up closing #347 so I was never added to the org. I supose adding me to the team then silently failed

I don't know if there's something that could be done to improve the process -- could the pulumi update process check that the user is actually a member in the org? Currently it seems to just silently fail to add the user to a team in that case
